web/src/app/admin/embeddings/interfaces.ts

```diff
@@ -9,19 +12,32 @@ export enum RerankerProvider {
   COHERE = "cohere",
 }
 export interface AdvancedDetails {
-  multilingual_expansion: string[];
+  model_name: string;
```

Comment:
This class name "AdvancedDetails" seems too generic.  could mean anything.

Reply:
Definitely agree looking back! Updating across the board to be clearer